docs/source/features/moe.rst

```diff
@@ -52,16 +52,18 @@ Other options include:
 1. ``moe_input_jitter_eps`` adds noise to the input tensor by applying jitter with a specified epsilon value.
 
 2. ``moe_token_dropping`` enables selectively dropping and padding tokens for each expert to achieve
-   a specified capacity.
+   a specified capacity, similar to GShard, Switch-Transformer, and DeepSpeed-MoE. Briefly, if the number
```

Comment:
fix run-on sentence.

moe_token_dropping enables selectively dropping and padding tokens for each expert to achieve a specified capacity. Similar to GShard, Switch-Transformer, and DeepSpeed-MoE. Briefly, if the number of tokens routed to an expert exceeds its capacity, then the exceeding tokens are dropped.